### ISSUE TITLE
[Feature Request] Allow css namespacing for vuetify theme

### ISSUE THREAD

**Issue by johannes-z** [vuetifyjs/vuetify#9454]
### Problem to solve
When using vuetify in an existing application (e.g. CMS), the installation of the styles can break existing styles of the page.

The only way right now to circumvent this, is providing the `disable: true` option for the `theme`, which disables the injection of the style in the page. The style can then be loaded into a css namespace with scss for example:

```scss
.my-namespace {
  @import "~vuetify/dist/vuetify.min";
}
```

The problem is, that specifing `disable: true` also ignores any provided options, most importantly colors. Also, this doesn't work anymore too: `options: { customProperties: true }`.

Because of this behavior, the vuetify stylesheet cannot be namespaced, because colors aren't working anymore.

`dist/vuetify.min.css` apparently only provides component styles, not colors. So you get buttons like this:

![image](https://user-images.githubusercontent.com/9069888/67470443-07c8e880-f64e-11e9-87d4-98a025179961.png)

### Proposed solution
`disable: true` should only disable stylesheet injection. Options like `customProperties: true` should still work, using the global css variables. CSS variables can be generated, even when `disable: true` is set, because they don't (usually) mess with global styles.

**Comment by wouerner** [comment]
I need it too! I have some problems with drupal 7.

**Comment by leafchild0** [comment]
I need this as well. Using Vaadin in the existing app. Adding Vuetify styles breaks most of the existing styling. The namespace will help a lot.
The proposed solution doesn't work.

**Comment by danielMonroy96** [comment]
Hi good day, 
Did any of you succeed?

**Comment by giddie** [comment]
No, not so far. So far I've managed using iframes where I need to embed vuetify. Not ideal, but it works OK and I suppose it does have the advantage of being able to keep the tech stacks separate.

**Comment by dapug** [comment]
This sure better be on the docket for v3.  Using Vuetify in an existing Bootstrap app is a must.

**Comment by J-Sek** [comment]
Preview should be inside the iframe in any serious CMS app - if your CSS won't leak, the JS will.

**Comment by eolamisan** [comment]
There is micro front end app use case. Probably others. We use workaround to unload css files by prefix.
